### PR TITLE
JP Onboarding: update card styles on authorize form

### PR DESCRIPTION
**Pull request by keoshi**
#### Changes proposed in this Pull Request

* Jetpack onboarding: update card styles on authorize form.

#### Testing instructions

* Fire up this PR.
* Spin up a [new JN site](https://jurassic.ninja/create/?shortlived&jetpack-beta).
* When prompted to connect, copy the URL from the “Set up Jetpack” button and append `&calypso_env=development` to it.
* Do this while logged in to WordPress.com and you should see the screen below. Make sure it all looks good.

#### Before

image

#### After

image

### CODE REVIEW

**Comment by shaunandrews** [comment]
:ship:

**Comment by jeffgolenski** [comment]
Also 🚢  @keoshi

File: client/jetpack-connect/authorize.js [hunk 1]
```diff
@@ -684,7 +684,7 @@ export class JetpackAuthorize extends Component {
 							siteIsOnSitesList={ this.props.isAlreadyOnSitesList }
 						/>
 						<AuthFormHeader authQuery={ this.props.authQuery } />
-						<Card>
+						<Card className="jetpack-connect__logged-card">
```
[the remark this answers is not in the document]
Comment (keoshi) [reply client/jetpack-connect/authorize.js]: Oh, copy/pasta fail! Thanks for catching that.

Fixed in https://github.com/Automattic/wp-calypso/pull/31837/commits/8614e362b2d6801cface52b7d5e75b2450da3f21

**Comment by keoshi** [comment]
Thanks, everyone!

**Pull request merged**